nautobot/core/templates/home.html

```diff
@@ -23,87 +23,95 @@
-                                                                {% autoescape off %}
-                                                                    {{ group_item_details.rendered_html }}
-                                                                {% endautoescape %}
-                                                            {% else %}
-                                                                <span class="badge pull-right">{{ group_item_details.count }}</span>
-                                                                <p style="padding-left: 20px;">
-                                                                    {% comment %}
-                                                                        Use 'url xxx as variable' so that an invalid
-                                                                        link doesn't throw a NoReverseMatch exception.
-                                                                    {% endcomment %}
-                                                                    {% url group_item_details.link as group_url %}
-                                                                    {% if group_url %}
-                                                                        <a href="{{ group_url }}">{{ group_item_name }}</a>
-                                                                    {% else %}
-                                                                        <a>ERROR: Invalid link!</a>
-                                                                    {% endif %}
-                                                                </p>
-                                                            {% endif %}
-                                                        {% else %}
-                                                            <span class="badge pull-right"><i class="mdi mdi-lock"></i></span>
-                                                            <p style="padding-left: 20px;">{{ group_item_name }}</p>
-                                                        {% endif %}
-                                                    {% endif %}
-                                                {% endfor %}
-                                            </div>
-                                        {% endif %}
-                                    {% endfor %}
-                                {% endif %}
+                                        {% endfor %}
+                                    {% endif %}
+                                </div>
                             </div>
-                        </div>
-                    {% endif %}
-                {% endfor %}
+                        {% endif %}
+                    {% endfor %}
+                </div>
             </div>
         </div>
     </div>
 {% endblock %}
+{% block javascript %}
+{{ block.super }}
+{% include 'inc/homepage_javascript.html' %}
```

Comment:
Using the new templatetag added in #4892, and avoiding the need for a separate `inc/` file that's only used once:

```suggestion
<script src="{% versioned_static 'js/homepage_layout.js' %}"
        onerror="window.location='{% url 'media_failure' %}?filename=js/homepagelayout.js'"></script>
```

Reply:
Done.  Thanks I didn't know this was possible.